packages/rest/package.json

```diff
@@ -1,13 +1,21 @@
 {
   "name": "@midna/rest",
-  "private": true,
-  "main": "./src/index.ts",
-  "types": "./src/index.ts",
-  "peerDependencies": {
-    "zod": "^3.22.2"
+  "version": "0.0.1",
+  "description": "Client for the Midna server",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc"
   },
+  "files": [
+    "/dist"
+  ],
   "dependencies": {
-    "@midna/db": "workspace:*",
-    "@ts-rest/core": "^3.30.2"
+    "@ts-rest/core": "^3.30.2",
+    "tslib": "^2.6.2",
+    "zod": "^3.22.2"
```

Comment:
zod can be listed as a peer dependency in all these packages. Many applications use it themselves, so describing it here might pull in duplicate versions